lbrynet/lbrynet_console/Console.py

```diff
@@ -366,11 +369,13 @@ def _setup_query_handlers(self):
         ]
 
         def get_blob_request_handler_factory(rate):
-            self.blob_request_payment_rate_manager = PaymentRateManager(self.session.base_payment_rate_manager,
-                                                                        rate)
-            handlers.append(BlobRequestHandlerFactory(self.session.blob_manager,
-                                                      self.session.wallet,
-                                                      self.blob_request_payment_rate_manager))
+            self.blob_request_payment_rate_manager = PaymentRateManager(
+                self.session.base_payment_rate_manager, rate
```

Comment:
another pycharm question: how do you get it to do alignment correctly? i feel like its really stupid about autoformatting long lines